cli/src/commands/lint.ts

```diff
@@ -33,7 +33,7 @@ export default class Lint extends Command {
     const lintingErrors = lint(contract);
 
     lintingErrors.forEach(error => {
-      this.error(error.message);
+      this.error(error.message, { exit: false });
```

Comment:
The `lint` command should exit with a non-0 exit code at the end of execution. This helps CI processes halt workflows.